src/vtbackend/Terminal.h

```diff
@@ -391,17 +391,9 @@ class Terminal
     void updateInputMethodPreeditString(std::string preeditString);
     // }}}
 
```

Comment:
Looks like we want to get rid of the inner lock then

Reply:
Deleted _innerLock